### PR TITLE
Lily

### PR DESCRIPTION
**Pull request by lilynnabeze**

### CODE REVIEW

File: src/components/Body.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  //     })
+  //     .catch((error) => {
+  //       setError(error.message);
+  //       setData(null);
+  //     })
+  //     .finally(() => {
+  //       setLoading(false);
+  //     });
+  // }, []);
+
```
Comment (tundeojediran) [inline src/components/Body.jsx]: remove commented code when not needed

File: src/components/Body.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        const response = await axios.get(`https://swapi.dev/api/films`);
+        setData(response.data);
+        // setError(null);
```
Comment (tundeojediran) [inline src/components/Body.jsx]: Any reason for commenting this line?

File: src/components/Body.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        setError(err.message);
+        setData(null);
+      } finally {
+        setLoading(false);
+      }
+    };
+    getData();
+  }, []);
+
+  console.log(data, "data");
```
Comment (tundeojediran) [inline src/components/Body.jsx]: remove logs as you only use them for debugging

File: src/components/Body.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  //     .finally(() => {
+  //       setLoading(false);
+  //     });
+  // }, []);
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        const response = await axios.get(`https://swapi.dev/api/films`);
+        setData(response.data);
```
Comment (tundeojediran) [inline src/components/Body.jsx]: You could do `setData(response.data.results)` here so you won't have to do optional chaining where you are making use of the data

**Review by tundeojediran — CHANGES_REQUESTED** [review]
Good job Asonwa. I added some comments for some areas you need to address before I approve your PR

File: src/components/MovieCard.jsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "../styles/MovieCard.css";
+
+function MovieCard({ data }) {
+  console.log(data);
```
Comment (tundeojediran) [inline src/components/MovieCard.jsx]: remove log

File: src/components/MovieCard.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  const CardItem = data?.results?.map(
+    ({ episode_id, title, release_date, opening_crawl }) => {
+      return (
+        <li key={episode_id} className="movie-card">
```
Comment (tundeojediran) [inline src/components/MovieCard.jsx]: Did you delete the background image implementation we did some time ago?

File: src/components/Body.jsx (hunk 2 above)
Comment (lilynnabeze) [reply src/components/Body.jsx]: That was when I was testing out the code, it seemed it wasn't doing anything even when commented out but I later figured out it's function.

File: src/components/Body.jsx (hunk 4 above)
Comment (lilynnabeze) [reply src/components/Body.jsx]: Oh great..thank you

**Pull request merged**